digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadNetworkDAO.scala

```diff
@@ -16,8 +16,8 @@ class RoadNetworkDAO {
 
   protected def logger = LoggerFactory.getLogger(getClass)
 
-  def createPublishedRoadNetwork: Unit = {
-    sqlu"""INSERT INTO published_road_network (id, created) VALUES (PUBLISHED_ROAD_NETWORK_SEQ.NEXTVAL, sysdate)""".execute
+  def createPublishedRoadNetwork (networkVersion: Long): Unit = {
```

Comment:
Small suggestion:

If we are to use Sequences.nextPublishedRoadNetworkId on every call of this, then why not define this as the default value?